apps/intellij/src/main/kotlin/dev/nx/console/toolWindow/NxExecutor.kt

```diff
@@ -0,0 +1,46 @@
+package dev.nx.console.toolWindow
+
+import com.intellij.execution.Executor
+import com.intellij.execution.RunManager
+import com.intellij.execution.RunnerAndConfigurationSettings
+import com.intellij.execution.executors.DefaultRunExecutor
+import com.intellij.execution.runners.ExecutionUtil
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+import dev.nx.console.run.NxCommandConfiguration
+import dev.nx.console.run.NxCommandConfigurationType
+import dev.nx.console.run.NxRunSettings
+
+class NxExecutor(val project: Project) {
```

Comment:
In general it's good to have a class like this, but:
- it should live under `dev.nx.console.run` because we will reuse it for other things (e.g. the run target actions)
- it should be named differently because `Executor` is already a name in Nx-context. Maybe `NxTaskExecutionManager`?
- it should have a different signature. Since you only use one project and one target, I would do it like `fun execute(nxProject: NxProject, nxTarget: NxTarget)`